locust/web.py

```diff
@@ -61,6 +62,10 @@ def swarm():
 
     locust_count = int(request.form["locust_count"])
     hatch_rate = float(request.form["hatch_rate"])
+    name = request.form["locustfile"]
```

Comment:
maybe we can change it to more readable variable name. how about if we change the name into the name of request form name like the others(line 63,64)? so the variable name should be `locustfile`